deploy-monitoring.sh

```diff
@@ -1,28 +1,36 @@
 #!/bin/bash
 
 # Change the namespace to that of your project
-NAMESPACE=myproject
+NAMESPACE=${1:-myproject}
+ISOCP=false
 
 # Get the current directory
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 
+# Test if running against an OpenShift cluster
+kubectl get route > /dev/null 2>&1
+[[ $? -eq 0 ]] && ISOCP=true
+
 # Prometheus / Alertmanager
 kubectl apply -f $DIR/monitoring/alerting-interconnect.yaml -n $NAMESPACE
-kubectl apply -f $DIR/monitoring/prometheus.yaml -n $NAMESPACE
+cat $DIR/monitoring/prometheus.yaml | sed "s/myproject/${NAMESPACE}/g" | kubectl apply -n $NAMESPACE -f -
```

Comment:
ah, that's going to mean my doc updates in https://github.com/rh-messaging/amq-docs/pull/1098 aren't necessary?

Reply:
@pwright Yes, the script changes will no longer require users to modify the namespace in other files anymore.